src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -2051,11 +2051,14 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
             tableName = table.name.split(".")[1];
         }
 
+        let seqName = `${tableName}_${columnName}_seq`;
+        if (seqName.length > 63) // note doesn't yet handle corner cases where .length differs from number of UTF-8 bytes
```

Comment:
instead of hard-coded magic value `63` I would recommend to use `Driver.maxAliasLength` property.

Reply:
I think in all other places we already use a `shorten` function to shorten things... Can we use it here as well to achieve consistency?

Reply:
I have pushed an amended commit that uses the `maxAliasLength` property. I have not used `shorten` because it doesn't match the truncation algorithm that Postgres uses, thus won't fix the issue.